component/authenticator/src/test/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticatorTest.java

```diff
@@ -0,0 +1,335 @@
+            + "s6XIGB9zYjfpnTDoea7TmjJ0Wk07JZBayyyqGrvsZyIf/JlKf/ITywZhtaRbxxlo\n"
+            + "bLU12i/RzYV/vh5awgbsKEr59EpmM8m4PRdhR5mN75VuusnpVfILShV92cMTBYyp\n"
+            + "v1eZZ9z8Wp8n3o3wtaAIlU2Tam17OcH7f5Hkz5XCTQIDAQABMA0GCSqGSIb3DQEB\n"
+            + "CwUAA4IBAQAGYqiugjx1Ak9o92xPr2wETtboyzHdDE6HYOlPDYtdh+gKZLb6MwSo\n"
+            + "2zr931l3eBq1wG/G48zW1+GC09MYq+DrWDO8ZM17q3xuZZdflRRHs4IvdjHX7K3w\n"
+            + "NNWNMUHeMlWEf8LufEOpw/3P3GH4i0SqZ7ld4nGlr2S3o4L6CY61qQCA6NmDZYHz\n"
+            + "ybs2v5i+rXLjFCaw734DlBRN6LGapEVLwznjYHJIsADLXLWlqvotG2psw0cYGJl5\n"
+            + "lnAa+yj6cL7kyF145DfB1QyUL4+tpEsiUBnFf6QaROEPhBZ9xl61zhg+6W8g+4q9\n" + "sPI+QGNS48ZDpXWJ8mXDTw0LoBuYKiKH";
+
+    private static final String CERT_WITH_SUBJECT_IN_THE_EMAIL = "MIIDyDCCArCgAwIBAgIJAMuvHuw9SZ+IMA0GCSqGSIb3DQEBCwUAMIGFMRMwEQYD\n"
+            + "VQQDDApjb21tb25OYW1lMQswCQYDVQQGEwJTTDEQMA4GA1UECAwHV2VzdGVybjEQ\n"
+            + "MA4GA1UEBwwHQ29sb21ibzENMAsGA1UECgwEV1NPMjELMAkGA1UECwwCUUExITAf\n"
+            + "BgkqhkiG9w0BCQEWEmJ1ZGRoaW1haEB3c28yLmNvbTAgFw0xOTA1MTYwNzU4MDha\n"
+            + "GA8zMDE4MDkxNjA3NTgwOFowgYUxEzARBgNVBAMMCmNvbW1vbk5hbWUxCzAJBgNV\n"
+            + "BAYTAlNMMRAwDgYDVQQIDAdXZXN0ZXJuMRAwDgYDVQQHDAdDb2xvbWJvMQ0wCwYD\n"
+            + "VQQKDARXU08yMQswCQYDVQQLDAJRQTEhMB8GCSqGSIb3DQEJARYSYnVkZGhpbWFo\n"
+            + "QHdzbzIuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAm8rszlo/\n"
+            + "t1BEZg7kcUt1LaAkbP2q1wFcf63SMOewMbOSVUr/48iE2KIG7l1fOBo7up2uA4N0\n"
+            + "TYNWfjheAtglvFMcxGUiixxlH/AikAdpO1ClNisvC+MMwPA3rQkjQToPlLvLTBD/\n"
+            + "+y9LuHmD/s0V15xGKlNHUoA2s1JGRmcUtWRK/XrxRcQAMRaf1mI72FUwlDAvzQYe\n"
+            + "amqlMIpsrzDMm7VSzleL2p3xjMPV9P0tHtCucL+THV35Y4ktVLpYbgLzGRUgPUCI\n"
+            + "GXEgbeubqMS7Dj011fFYmHzM8CB+oHbW+mpx/KyD2HuVsRKOAxKwBev0kprmpu18\n"
+            + "vwLrOGltlfOM4QIDAQABozcwNTAzBgNVHREELDAqgg53d3cuZ29vZ2xlLmNvbYIJ\n"
+            + "d3d3LmFjLmxrgg13d3cubXJ0LmFjLmxrMA0GCSqGSIb3DQEBCwUAA4IBAQCOhFvY\n"
+            + "NmdW+PKwiHBD486MATJSfXXbOxkdB1OTNx++iWEdOupnTmQa9sPMA5dfCrsehf04\n"
+            + "RtERe4LbSJoug3T4OIww2/6kRxP52+zZnwpg7GgMFN7RMgUXE9ptHC0mOnIL+LpC\n"
+            + "vD9CDN2WBvnCHBA/x21Q+fTjchDNYG96YLJn2uW6pwPuRAzSBFpUmko6KMUgqUVJ\n"
+            + "MnL4jtEI7tHs89GB+tfurD2dsSLW5ghXaDwmZTNuaUOgD8SRYwh0AG+en1Xk2v3/\n"
+            + "73zDq+0+CCuZv87EyQbA5QobwBlYNe45ocyxSzocJLTapVkcXytDr/+ZhhdB7ybL\n" + "UZoGqB7ayed6KBFi";
+
+    @DataProvider(name = "provideX509Certificates")
+    public Object[][] provideTestData() throws Exception {
+        CertificateFactory factory = CertificateFactory.getInstance("X.509");
+        X509Certificate cert1 = (X509Certificate) factory
+                .generateCertificate(new ByteArrayInputStream(DatatypeConverter.parseBase64Binary(
+                        CERT_WITH_NO_ALTERNATIVE_NAMES)));
+        X509Certificate obj1[] = { cert1, null };
```

Comment:
Let's use the following declaration format for better readability.
```suggestion
        X509Certificate[] obj1 = { cert1, null };
```
Also, it would be great if these variables had more meaningful names.